mlir/include/mlir/Dialect/XeGPU/Transforms/Transforms.h

```diff
@@ -14,11 +14,67 @@ class RewritePatternSet;
 
 namespace xegpu {
 
+/// Options to control the XeGPU unrolling. Its main purpose is to
+/// provide a way to customize the native shape of the operation.
+struct UnrollOptions {
+  using FilterConstraintFnType = std::function<LogicalResult(Operation *op)>;
+  /// Callback function that indicates whether vector unrolling should be
```

Comment:
nit: let's place this comment above "using" to have uniform look :)

Reply:
fixed it